versioned_docs/version-24.04/administration/centreon-ha/operating-guide.md

```diff
@@ -0,0 +1,800 @@
+    * snmptrapd (systemd:snmptrapd):     Started central1
+
+Migration Summary:
+```
+
+> These commands should return no errors. If there are "Failed actions" on any resource, troubleshoot them using the [troubleshooting guide](troubleshooting-guide.md).
+
+### Using the Centreon interface
+
+The installation process includes the monitoring of the members of the cluster by a poller. This way, you can be notified if a member of the cluster goes down.
+
+The Resource Status page gives you the following information:
+
+* On both central nodes, the **PCS-Status** service gives you the detailed state of the cluster. The output of the service in the details panel is the output of the `pcs status` command.
+* You can know which central node is the active node by looking at which node is carrying the cluster resources in the output of the **PCS-Status** service on each central.
+
+## Check the constraints
+
+If a failover has occurred at some point, there may be some leftover location constraints. Run the following command to display the current constraints:
+
+```bash
+pcs constraint
+```
+
+The command should return this:
+
+```text
+Location Constraints:
+  Resource: cbd_rrd-clone
+    Disabled on:
+      Node: @DATABASE_NODE1_NAME@ (score:-INFINITY)
+      Node: @DATABASE_NODE2_NAME@ (score:-INFINITY)
+  Resource: centreon
+    Disabled on:
+      Node: @DATABASE_NODE1_NAME@ (score:-INFINITY)
+      Node: @DATABASE_NODE2_NAME@ (score:-INFINITY)
+  Resource: ms_mysql-clone
+    Disabled on:
+      Node: @CENTRAL_NODE1_NAME@ (score:-INFINITY)
+      Node: @CENTRAL_NODE2_NAME@ (score:-INFINITY)
+  Resource: php-clone
+    Disabled on:
+      Node: @DATABASE_NODE1_NAME@ (score:-INFINITY)
+      Node: @DATABASE_NODE2_NAME@ (score:-INFINITY)
+Ordering Constraints:
+Colocation Constraints:
+  vip_mysql with ms_mysql-clone (score:INFINITY) (rsc-role:Started) (with-rsc-role:Master)
+  ms_mysql-clone with vip_mysql (score:INFINITY) (rsc-role:Master) (with-rsc-role:Started)
+Ticket Constraints:
+
+```
+
+The output shows the constraints you have defined during the installation procedure: the **ms_mysql-clone** resource only runs on the database nodes, the **cbd_rrd-clone**, **centreon** and **php-clone** resources only run on the central nodes.
+
+To remove unwanted constraints, run the following command:
+
+```bash
+pcs resource clear centreon
+```
+
+## Check the status of the database synchronization
+
+To check that the database synchronization is working, run the following command:
+
+```bash
+/usr/share/centreon-ha/bin/mysql-check-status.sh
+```
+
+The command should return the following information:
+
+```bash
+Connection MASTER Status '@DATABASE_NODE1_NAME@' [OK]
+Connection SLAVE Status '@DATABASE_NODE2_NAME@' [OK]
+Slave Thread Status [OK]
+Position Status [OK]
+```
+
+> If the synchronization shows `KO`, you must fix it with the help of the [operating guide](operating-guide.md).
```

Comment:
This sentence references to this exact page. Is it useful? I guess it should be rephrased...

Reply:
I see you've commented this out for the moment. I'm not sure there's one unique solution in case of a database issue; at least in my distant past, it was sometimes a bit tricky to get out of this. At the moment, I would recommend reaching out to an expert...

Reply:
@itoussies what do you think?

Reply:
Sorry for the late answer. Yeah as Vincent said, it's a bit tricky, a failed synchronization can have many causes. The least we can say is how to reinstate the replication as described in the installation procedure.

Reply:
I added the repair procedure.